edge-hub/core/src/Microsoft.Azure.Devices.Edge.Hub.Http/controllers/RegistryController.cs

```diff
@@ -513,10 +522,18 @@ async Task<bool> AuthenticateAsync(string deviceId, Option<string> moduleId, Opt
             return false;
         }
 
-        async Task SendResponseAsync(HttpStatusCode status, string jsonContent = "")
+        async Task SendResponseAsync(HttpStatusCode status, string jsonContent = "", Dictionary<string, string> headers = null)
         {
             this.Response.StatusCode = (int)status;
 
+            if (headers != null)
```

Comment:
>if (headers != null) [](start = 12, length = 20)

the team doesn't like null checks, you can avoid this by having an overload instead that takes (HttpStatus, string), and calls this with an empty Dictionary instead of null.

Reply:
fixed